XCTWebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -26,10 +26,10 @@ + (NSArray *)routes
     [[FBRoute POST:@"/session"] respond:^ id<FBResponsePayload> (FBRouteRequest *request) {
 
       NSDictionary *requirements = request.arguments[@"desiredCapabilities"];
-      NSString *bundleID = requirements[@"CFBundleIdentifier"];
-      NSString *appPath = requirements[@"FBBinaryPath"];
-      NSAssert(bundleID, @"Should have bundle ID");
-      NSAssert(appPath, @"Should have app path");
+        NSString *bundleID = requirements[@"bundleId"];
+        NSString *appPath = requirements[@"app"];
```

Comment:
I agree that appPath is better, but the spec would have to be changed.
The reason is that with `app` you can specify a path to an app file, a url
to download the app from, or the bundleId of the app already installed on
the device.
https://code.google.com/p/selenium/source/browse/spec-draft.md?repo=mobile

On Thu, Nov 12, 2015 at 3:56 AM, Marek Cirkos <notifications@github.com>
wrote:

> In XCTWebDriverAgentLib/Commands/FBSessionCommands.m
> <https://github.com/facebook/WebDriverAgent/pull/46#discussion_r44648822>:
>
> > @@ -26,10 +26,10 @@ + (NSArray *)routes
> >      [[FBRoute POST:@"/session"] respond:^ id<FBResponsePayload> (FBRouteRequest *request) {
> >
> >        NSDictionary *requirements = request.arguments[@"desiredCapabilities"];
> > -      NSString *bundleID = requirements[@"CFBundleIdentifier"];
> > -      NSString *appPath = requirements[@"FBBinaryPath"];
> > -      NSAssert(bundleID, @"Should have bundle ID");
> > -      NSAssert(appPath, @"Should have app path");
> > +        NSString *bundleID = requirements[@"bundleId"];
> > +        NSString *appPath = requirements[@"app"];
>
> Isn't this too ambiguous. Maybe appPath would be better?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/WebDriverAgent/pull/46/files#r44648822>.
>